static/js/chart/draw_choropleth.ts

```diff
@@ -100,15 +100,23 @@ function getColorScale(
   const extent = d3.extent(Object.values(dataValues));
   const medianValue = d3.median(Object.values(dataValues));
   let domainValues: number[] = domain || [extent[0], medianValue, extent[1]];
-  if (statVar.indexOf("Temperature") >= 0) {
+  if (
+    statVar.indexOf("Temperature") >= 0 ||
+    statVar.indexOf("Number of Months Based on") >= 0
+  ) {
+    // SV Name of "Wet Bulb Temps"
     let range: any[] = [
       d3.interpolateBlues(1),
       d3.interpolateBlues(0.8),
       MIN_COLOR,
       d3.interpolateReds(0.8),
       d3.interpolateReds(1),
     ];
-    if (statVar.indexOf("Difference") >= 0) {
+
+    if (statVar.indexOf("Number of Months Based on") >= 0) {
+      // SV Name of "Wet Bulb Temps"
+      domainValues = domain || [0, 15];
```

Comment:
nit: these curated domains (eg. [0, 15], [-10, -5, 0, 5, 10]) could be constants

Reply:
done